diff --git a/examples/rn-connection-and-error/frontend/android/app/build.gradle b/examples/rn-connection-and-error/frontend/android/app/build.gradle
index b4584d7ae..bb923747c 100644
--- a/examples/rn-connection-and-error/frontend/android/app/build.gradle
+++ b/examples/rn-connection-and-error/frontend/android/app/build.gradle
@@ -10,7 +10,7 @@ react {
     //   The root of your project, i.e. where "package.json" lives. Default is '..'
     // root = file("../")
     //   The folder where the react-native NPM package is. Default is ../node_modules/react-native
-    // reactNativeDir = file("../node_modules/react-native")
+    reactNativeDir = file("../../../../../node_modules/react-native")
     //   The folder where the react-native Codegen package is. Default is ../node_modules/@react-native/codegen
     // codegenDir = file("../node_modules/@react-native/codegen")
     //   The cli.js file which is the React Native CLI entrypoint. Default is ../node_modules/react-native/cli.js
@@ -120,4 +120,4 @@ dependencies {
     }
 }

-apply from: file("../../node_modules/@react-native-community/cli-platform-android/native_modules.gradle"); applyNativeModulesAppBuildGradle(project)
+apply from: file("../../../../../node_modules/@react-native-community/cli-platform-android/native_modules.gradle"); applyNativeModulesAppBuildGradle(project)
diff --git a/examples/rn-connection-and-error/frontend/android/settings.gradle b/examples/rn-connection-and-error/frontend/android/settings.gradle
index d8bb2b9ec..d33adf743 100644
--- a/examples/rn-connection-and-error/frontend/android/settings.gradle
+++ b/examples/rn-connection-and-error/frontend/android/settings.gradle
@@ -1,4 +1,4 @@
 rootProject.name = 'ExampleRnConnectionAndError'
-apply from: file("../node_modules/@react-native-community/cli-platform-android/native_modules.gradle"); applyNativeModulesSettingsGradle(settings)
+apply from: file("../../../../node_modules/@react-native-community/cli-platform-android/native_modules.gradle"); applyNativeModulesSettingsGradle(settings)
 include ':app'
-includeBuild('../node_modules/@react-native/gradle-plugin')
+includeBuild('../../../../node_modules/@react-native/gradle-plugin')
diff --git a/examples/rn-connection-and-error/frontend/ios/ExampleRnConnectionAndError.xcodeproj/project.pbxproj b/examples/rn-connection-and-error/frontend/ios/ExampleRnConnectionAndError.xcodeproj/project.pbxproj
index 689407bca..fd35ccd42 100644
--- a/examples/rn-connection-and-error/frontend/ios/ExampleRnConnectionAndError.xcodeproj/project.pbxproj
+++ b/examples/rn-connection-and-error/frontend/ios/ExampleRnConnectionAndError.xcodeproj/project.pbxproj
@@ -264,7 +264,7 @@
 			);
 			runOnlyForDeploymentPostprocessing = 0;
 			shellPath = /bin/sh;
-			shellScript = "set -e\n\nWITH_ENVIRONMENT=\"../node_modules/react-native/scripts/xcode/with-environment.sh\"\nREACT_NATIVE_XCODE=\"../node_modules/react-native/scripts/react-native-xcode.sh\"\n\n/bin/sh -c \"$WITH_ENVIRONMENT $REACT_NATIVE_XCODE\"\n";
+			shellScript = "set -e\n\nWITH_ENVIRONMENT=\"../../../../node_modules/react-native/scripts/xcode/with-environment.sh\"\nREACT_NATIVE_XCODE=\"../../../../node_modules/react-native/scripts/react-native-xcode.sh\"\n\n/bin/sh -c \"$WITH_ENVIRONMENT $REACT_NATIVE_XCODE\"\n";
 		};
 		00EEFC60759A1932668264C0 /* [CP] Embed Pods Frameworks */ = {
 			isa = PBXShellScriptBuildPhase;
@@ -601,7 +601,7 @@
 					"-DFOLLY_USE_LIBCPP=1",
 				);
 				OTHER_LDFLAGS = "$(inherited)";
-				REACT_NATIVE_PATH = "${PODS_ROOT}/../../node_modules/react-native";
+				REACT_NATIVE_PATH = "${PODS_ROOT}/../../../../../node_modules/react-native";
 				SDKROOT = iphoneos;
 			};
 			name = Debug;
@@ -671,7 +671,7 @@
 					"-DFOLLY_USE_LIBCPP=1",
 				);
 				OTHER_LDFLAGS = "$(inherited)";
-				REACT_NATIVE_PATH = "${PODS_ROOT}/../../node_modules/react-native";
+				REACT_NATIVE_PATH = "${PODS_ROOT}/../../../../../node_modules/react-native";
 				SDKROOT = iphoneos;
 				VALIDATE_PRODUCT = YES;
 			};
diff --git a/examples/rn-connection-and-error/frontend/metro.config.js b/examples/rn-connection-and-error/frontend/metro.config.js
index 0bc527118..7be8bfb2f 100644
--- a/examples/rn-connection-and-error/frontend/metro.config.js
+++ b/examples/rn-connection-and-error/frontend/metro.config.js
@@ -16,14 +16,6 @@
 //
 ////////////////////////////////////////////////////////////////////////////

-const {getDefaultConfig, mergeConfig} = require('@react-native/metro-config');
-
-/**
- * Metro configuration
- * https://facebook.github.io/metro/docs/configuration
- *
- * @type {import('metro-config').MetroConfig}
- */
-const config = {};
-
-module.exports = mergeConfig(getDefaultConfig(__dirname), config);
+module.exports = require("@realm/metro-config")({
+  projectRoot: __dirname,
+});
diff --git a/examples/rn-connection-and-error/frontend/package.json b/examples/rn-connection-and-error/frontend/package.json
index 4e951d84c..2dd005898 100644
--- a/examples/rn-connection-and-error/frontend/package.json
+++ b/examples/rn-connection-and-error/frontend/package.json
@@ -22,6 +22,7 @@
     "@babel/runtime": "^7.20.0",
     "@react-native/eslint-config": "^0.72.2",
     "@react-native/metro-config": "^0.72.11",
+    "@realm/metro-config": "*",
     "@tsconfig/react-native": "^3.0.0",
     "@types/react": "^18.0.24",
     "@types/react-test-renderer": "^18.0.0",
